app/controllers/admin/dashboard_controller.rb

```diff
@@ -11,4 +16,28 @@ def prep_calendar
     @division = current_division
     @calendar_events_url = "/admin/calendar_events?person_id=#{@person.id}"
   end
+
+  def prep_projects_grid
+    # Projects belonging to the current user
+    # 15 most recent projects, sorted by created date, then updated date
+    @recent_projects = @person.agent_projects.order(created_at: :desc, updated_at: :desc).limit(15)
+
+    @recent_projects_grid = initialize_grid(
+      @recent_projects,
+      include: [:primary_agent, :secondary_agent],
+      per_page: 15,
+      name: "recent_projects",
+      enable_export_to_csv: false
+    )
+
+    @status_filter_options = status_filter_options
+  end
+
+  def status_filter_options
+    filter_options = []
+    STATUS_FILTERS.each do |item|
+        filter_options.push([I18n.t("dashboard.status_options.#{item}"), item])
+    end
+    return filter_options
```

Comment:
Try to remember to use blocks and `map` for things like this.

`STATUS_FILTERS.map { |f| [I18n.t("dashboard.status_options.#{f}"), f] }`

And since it's so simple now you really don't even need the separate function...